global.json

```diff
@@ -0,0 +1,5 @@
+{
+ "sdk": {
+   "version": "3.0.100-preview8-013656"
```

Comment:
Please do not resolve a comment without pushing the fix for it.

Reply:
I removed it when he suggested and added this file to gitignore since I need this file to run in my environment. But when I chose commit suggested PR changes on github my local is behind the remote so I git pull. It seems that the git pull overwrote my gitignore and pushed this file again. This is a lesson learned.